docs/deploy/security_groups.md

```diff
@@ -1,32 +1,53 @@
-# Security groups
+# Security Groups in AWS Load Balancers
 
-The AWS Load Balancer Controller classifies security groups into two categories: frontend and backend.
+Leverage security groups to limit client connections to your load balancers, and tightly define connections with nodes. The AWS Load Balancer Controller defines two classifications of security groups: **frontend** and **backend**.
+
+- **Frontend Security Groups:** Determine the clients that can access the load balancers.
+- **Backend Security Groups:** Regulate the traffic flowing between load balancers and their associated EC2 instances or ENIs.
 
 ## Frontend Security Groups
 
-Frontend security groups control which clients can access the load balancers. The frontend security groups can be configured with the `alb.ingress.kubernetes.io/security-groups` annotation on Ingress resources or `service.beta.kubernetes.io/aws-load-balancer-security-groups` annotation on Service resources. If the annotations are not specified, the LBC will create one security group per load balancer, allowing traffic from `inbound-cidrs` to `listen-ports`.
+Frontend security groups govern access to load balancers by specifying which clients can connect. 
+
+### Configuration
+
+- Utilize the `alb.ingress.kubernetes.io/security-groups` [annotation on Ingress resources](../guide/ingress/annotations#annotations) for specification.
+- Alternatively, employ the `service.beta.kubernetes.io/aws-load-balancer-security-groups` [annotation on Service resources](../guide/service/annotations.md#annotations) to configure.
+
+### Default Behavior
+
+- If annotations are omitted, the Load Balancer Controller (LBC) initializes one security group for each load balancer.
+- The default setup permits traffic from `inbound-cidrs` to `listen-ports`.
 
 ## Backend Security Groups
 
-A single shared backend security group controls the traffic between load balancers and their target EC2 instances or ENIs. This security group is attached to the load balancers and is used as the traffic source in the ENI/Instance security group rules. The backend security group is shared between multiple load balancers.
+Backend Security Groups serve as the control mechanism for traffic between AWS Load Balancers and their target EC2 instances or ENIs. A single shared backend security group is utilized to manage the traffic.
```

Comment:
bring back `This security group is attached to the load balancers and is used as the traffic source in the ENI/Instance security group rules.`